core/templates/components/state-editor/state-editor-properties-services/state-skill.service.ts

```diff
@@ -32,6 +32,10 @@ export class StateLinkedSkillIdService extends StatePropertyService<string> {
     super(alertsService, utilsService);
     this.setterMethodKey = 'saveLinkedSkillId';
   }
+
+  get onStateLinkedSkillIdInitialized(): EventEmitter<string> {
```

Comment:
*Ditto

Reply:
Not sure if a meaningful test can be written for this getter so I haven't made any changes.